### DISCUSSION TITLE
Keyboard "left" Predict "middle"

### DISCUSSION THREAD

**Discussion by sofia-bg** [UNC-Project-Open-AAC/OS-DPI#216] [Help]
I am working on a solution ([KB_left_predict_middle](https://github.com/UNC-Project-Open-AAC/OS-DPI/files/12469584/kb_thirds_v1.zip)) where the first third of screen is prioritized as that is the easiest area for the user to access. I played around with just alphabet for typing and middle blank, but wanted to see about putting word prediction in. 

I have it set to fill, so I have the arrows at the bottom. I am wondering if there is anyway to get rid of them? So that the user would just get 4 or 5 or however many rows ultimately we end up deciding on?  Ideally that would allow me to use that space to make the space, delete, and clear buttons bigger (i.e., longer)

Also, I could not figure out how to make them blank to start. Init with $predict = ''  did not work. I just added five rows with a space in each at the top of the sheet as a solution for now... 

<img width="1511" alt="Screenshot 2023-08-29 at 5 25 12 PM" src="https://github.com/UNC-Project-Open-AAC/OS-DPI/assets/92817124/7f73ffe2-7ffc-447a-b151-28b63958a66b">

I also made a tab control version where the keyboard goes then to predict, keeping the middle a gap... obvious problems here as it prevents you from continuing to type. But was just playing around. If anyone has other ideas of how to keep everything in that third would be great. I am avoiding adding a row of word prediction across the top since it is already getting tight. 

[KB_left_tabControl_predict](https://github.com/UNC-Project-Open-AAC/OS-DPI/files/12469637/kb_thirds_v2.zip)

**Comment by gbishop** [comment]
You can hide those buttons by adding

`.page-control { display: none; }`

to your Customize control but there will be a blank space there instead. If I included the grid name on its top level you could customize its height to hide the missing bit. I probably should do that.

Or we could make those buttons optional with another checkbox in the grid configuration. 

You could make the "empty" value for $predict be something like $, then test for that in the actions.
